app/js/components/LabResultsList.jsx

```diff
@@ -0,0 +1,263 @@
+    }));
+    return (
+      <div className="lab-results-list">
+        <SortableTable
+          data={labResults}
+          columnMetadata={columnMetadata}
+          filteredFields={fields}
+          filterType="none"
+          showFilter={false}
+          isSortable={false}
+          rowOnClick={this.handleShowResultsEntryPage}
+          noDataMessage="No orders found"
+          defaultPageSize={10}
+          subComponent={(row) => {
+            const isPanel = (row.original.order.concept.set) && (row.original.status !== "Ordered");
+            const rowFields = ["TYPE", "RESULT", "NORMAL RANGE"];
+            const rowColumnMetadata = rowFields.map(columnName => ({
+              accessor: "",
+              Cell: data => <Cell {...data} columnName={columnName} type="panel" />,
+              className: `lab-results-list-cell-${columnName.replace(' ', '-').toLocaleLowerCase()}`,
+              headerClassName: 'lab-results-list-header',
+            }));
+            if (isPanel) {
+              return (
+                <div className="collapsible-panel">
+                  <SortableTable
+                    data={row.original.encounter.obs[0].groupMembers}
+                    columnMetadata={rowColumnMetadata}
+                    collapseOnDataChange={false}
+                    collapseOnPageChange={false}
+                    showPagination={false}
+                    defaultPageSize={row.original.encounter.obs[0].groupMembers.length}
+                    defaultClassName=""
+                  />
+                </div>
+              );
+            }
+            return '';
+          }}
+        />
+      </div>
+    );
+  }
+
+  render() {
+    const { patients } = this.props;
+    const { patientUUID } = this.state;
+    const selectedPatient = patients[patientUUID] || {};
+    const { encounters = [], orders = [] } = selectedPatient;
+
+    const getPatientLabResults = () => {
+      const labResults = orders.map((order) => {
+        const status = 'Ordered';
+        const matchedEnocunter = encounters.filter((encounter) => {
+          const testOrderObs = encounter.obs.filter(item => item.display.includes('Test order number:'));
+          const orderNumber = testOrderObs[0].value;
+          return orderNumber === order.orderNumber;
+        });
+        const hasEncounter = !R.isEmpty(matchedEnocunter);
+        if (hasEncounter) {
+          const encounter = matchedEnocunter[0];
+          const hasObs = !R.isEmpty(encounter.obs);
+          if (hasObs) {
+            
+            return {
+              order,
+              encounter: {
+                ...encounter,
+                obs: R.pipe(
+                  R.filter(item => !item.display.includes('Date of test results')),
+                  R.filter(item => !item.display.includes('Location of laboratory')),
+                  R.filter(item => !item.display.includes('Test order number')),
```

Comment:
Same comment as above... we shouldn't match on the display string, as it is will be pretty brittle... can we change this to match on item.concept.uuid?